protobuf/protoc-gen-govalidator/plugin/plugin.go

```diff
@@ -77,6 +77,20 @@ func genValidateFunc(g *protogen.GeneratedFile, msg *protogen.Message) {
 		repeated := f.Desc.Cardinality() == protoreflect.Repeated
 
 		if hasExt && repeated {
+			if v := getIntExtention(f, validator.E_RepeatedLenGte); v >= 0 {
```

Comment:
Do we want to validate the GTE value is less than the LTE value (if both value exists)?

For e.g., is this valid?
```
      (validator.repeated_len_gte) = 5,
      (validator.repeated_len_lte) = 2
```

Reply:
hhmm.. I think that makes sense but I live that responsibility to whoever is defining the proto files to make sure the right combination is used so we don't complicate plugin code. Worst case scenario is that one would fail first. `length_gte` and `length_lte` also has the same.